src/language-js/printer-estree.js

```diff
@@ -2560,7 +2560,11 @@ function printPathNoParens(path, options, print, args) {
               isBinaryish(n.expressions[i])
             ) {
               printed = concat([
-                indent(concat([parenLine, printed])),
+                // If we are using parenSpace, parenLine already contains a space, so this
+                // will be equivalent to `" " + printed + " "`
+                // If we are not using parenSpace, then parenSpace is an empty string, so
+                // this will be equivalent to `"" + printed + ""`
+                indent(concat([parenLine, printed, parenSpace])),
                 softline,
```

Comment:
Let's go with the approach (that I mangled on the 2.0.4 branch) from `1.19.1`:

https://github.com/Automattic/wp-prettier/blob/da6be4794765d435c2f42a3218545cca679a0454/src/language-js/printer-estree.js#L2526-L2529

Reply:
Done